src/tests/unit/tests/bug-filing/azure-boards/azure-boards-settings-form.test.tsx

```diff
@@ -1,38 +1,39 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { shallow } from 'enzyme';
-import * as React from 'react';
-import { IMock, Mock, MockBehavior, Times } from 'typemoq';
-
 import { Dropdown } from 'office-ui-fabric-react/lib/Dropdown';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import * as React from 'react';
```

Comment:
I find it odd that tslint rule to alphabetize imports let us do this kind of reordering :S

Reply:
Agreed. This just looks weird to me honestly...